build.gradle.kts

```diff
@@ -8,6 +8,7 @@ plugins {
   id("org.jetbrains.kotlin.jvm")
   `kotlin-dsl`
   groovy
+  id("com.google.devtools.ksp") version "1.5.31-1.0.0"
```

Comment:
Would I remove this plugin once this todo was resolved?
```
// TODO switch to moshi-kotlin-codegen (where this was upstreamed to) once updated to Moshi 1.13.0
```

Reply:
Nope you would just switch the Moshi-KSP dep below with mainline Moshi code gen instead. Though if you can go to kotlin 1.6 in that case, you could replace this with Moshi-IR instead and remove KSP all together